src/simplicial-hott/09-yoneda.rzk.md

```diff
@@ -345,53 +370,55 @@ contractible.
   := (x : A) -> is-contr (hom A a x)
 ```
 
+Initial objects satisfy an induction principle relative to covariant families.
+
 ```rzk
 #section initial-evaluation-equivalence
 
 #variable A : U
 #variable a : A
-#variable a-is-initial : is-initial A a
+#variable is-initial-a : is-initial A a
 #variable C : A -> U
-#variable C-is-covariant : is-covariant A C
+#variable is-covariant-C : is-covariant A C
```

Comment:
Same question as earlier. I think a syntax for "variable blocks" would be nice. @fizruk